crossfire/clients/occurrences.py

```diff
@@ -164,3 +179,58 @@ def __call__(self):
             return GeoDataFrame(self.data)
 
         return self.data
+
+
+def _flatten_df(data, nested_columns):
+    def _flatten_col(row, column_name):
+        column_data = row[column_name]
+        return Series(
+            {
+                f"{column_name}_{key}": value
+                for key, value in column_data.items()
+            }
+        )
+
+    keys = set(data.columns) & nested_columns
+    for key in keys:
+        data = concat(
+            [
+                data.drop(key, axis=1),
+                data.apply(_flatten_col, args=(key,), axis=1),
+            ],
+            axis=1,
+        )
+    return data
+
+
+def _flatten_list(data, nested_columns):
+    keys = set(data[0].keys()) & nested_columns
+    for item in data:
+        for key in keys:
+            if key in item:
+                value = item.get(key)
+                if value:
+                    item.update(
+                        {f"{key}_{k}": v for k, v in value.items() if v}
+                    )
```

Comment:
Try to keep the [flat path to success](https://www.vinta.com.br/blog/flat-success-path)


```suggestion
            if key not in item:
                continue
                
            value = item.get(key)
            if not value:
                continue

            item.update({f"{key}_{k}": v for k, v in value.items() if v})
```